recipes/squeakr/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "squeakr" %}
+{% set version = "0.5" %}
+{% set md5 = "311dd917960e91e4fb7fee34970cd0b5" %}
+
+package:
+  name : {{ name|lower }}
+  version : {{ version }}
+
+source:
+  url : https://github.com/splatlab/squeakr/archive/v{{ version }}.tar.gz
+  fn : {{ name|lower }}_{{ version }}.tar.gz
+  md5 : {{ md5 }}
+
+build:
+  number : 1
+
+requirements:
+  build:
+    - gcc  # [not osx]
+    - llvm  # [osx]
+    - bzip2
+    - zlib
+    - boost 
+    - openssl
+  run:
+    - bzip2
+    - zlib
+    - boost 
+    - openssl
```

Comment:
You need to add `- libgcc  # [not osx]` as runtime dependency.

Reply:
looks like this does the trick.  I will push the fix as soon as the `{{CONDA_BZIP2}}` variable is merged.